docs/DeveloperGuide.md

```diff
@@ -20,18 +197,18 @@
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
```

Comment:
Perhaps you could elaborate more on the user stories so that the rest can better understand the motivations of this application and how it caters to the target users.